sendto/plugins/caja-burn/Makefile.am

```diff
@@ -1,20 +1,32 @@
-plugindir = $(libdir)/caja-sendto/plugins
+NULL =
+
+AM_CPPFLAGS =					\
+	$(DISABLE_DEPRECATED_CPPFLAGS)		\
+	-DMATELOCALEDIR=\""$(datadir)/locale"\"	\
+	-I$(top_builddir)			\
+	-I$(top_srcdir)/sendto			\
+	$(NULL)
 
-AM_CPPFLAGS =                                      \
-        -DDATADIR=\"$(datadir)\"                \
-        -DICONDIR=\"$(icondir)\"                \
-	-DLOCALEDIR="\"$(datadir)/locale\""	\
-        -I$(top_srcdir)/sendto                  \
-        -I$(top_builddir)                       \
-        -I$(srcdir)/../				\
-        $(SENDTO_CFLAGS)               \
+AM_CFLAGS =					\
 	$(WARN_CFLAGS)				\
-	$(DISABLE_DEPRECATED)                   \
-	$(GIO_CFLAGS)
+	$(SENDTO_CFLAGS)			\
+	$(NULL)
+
+plugindir = $(libdir)/caja-sendto/plugins
 
 plugin_LTLIBRARIES = libnstburn.la
 
 libnstburn_la_SOURCES = caja-burn.c
+libnstburn_la_CPPFLAGS =			\
+	$(AM_CPPFLAGS)				\
+	-DDATADIR=\"$(datadir)\"		\
+	-DICONDIR=\"$(icondir)\"		\
+	-I$(srcdir)				\
```

Comment:
Is that actually useful?  I would be surprised you actually have to do that

Reply:
```shell
$ find . -type f -name Makefile.am -exec sed -i '/-I\$(srcdir)/d' {} \;
$ ./autogen.sh --prefix=/usr
$ make
<cut>
caja-burn.c:27:10: fatal error: nst-common.h: No such file or directory
   27 | #include "nst-common.h"
      |          ^~~~~~~~~~~~~~
compilation terminated.
<cut>
```